rpcs3/Emu/Cell/Modules/cellMsgDialog.cpp

```diff
@@ -391,7 +391,7 @@ error_code cellMsgDialogClose(f32 delay)
 	{
 		if (auto dlg = manager->get<rsx::overlays::message_dialog>())
 		{
-			thread_ctrl::spawn("cellMsgDialogClose() Thread", [=]
+			new named_thread("cellMsgDialogClose() Thread", [=]
```

Comment:
Stop it. Stop silencing warnings you don't understand, you can't fix it like here.